karapace/typing.py

```diff
@@ -18,4 +18,7 @@
 Subject = NewType("Subject", str)
 Version = Union[int, str]
 ResolvedVersion = NewType("ResolvedVersion", int)
+# note: the SchemaID is a unique id among all the schemas (and each version should be assigned to a different id)
+# basically the mapping SchemaId -> TypedSchema is an ImmutableMapping.
```

Comment:
during the development together with @giuseppelillo we wondered if the SchemaId was a "real" global id or an id valid only on the same user. I though that this info should be available during the definition of the type, I'm not referring to any specific mapping, here I'm just saying that it's a mapping in the mathematical term, it's a bidirectional function (an isomorphism) aka a mapping that do not vary since those objects represent values

Reply:
resolved OOG (out of github) :D